### PR TITLE
Changed executable name to match snap name on Snap Store, fixed typo …

### PR DESCRIPTION
**Pull request by julian-alarcon**
…on description

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "linux": {
       "category": "Network;Chat",
       "packageCategory": "net",
-      "executableName": "teams",
+      "executableName": "teams-for-linux",
```
Comment (IsmaelMartinez) [inline package.json:46]: It seems a reasonable change but, if someone is using the other executable name, they might complain a bit... 

If deciding to got ahead, change also the reference in this readme (https://github.com/IsmaelMartinez/teams-for-linux/blob/master/app/config/README.md)

Comment (benyanke) [reply package.json:46]: I think this is a logical change, despite there being some inertia here. I'd vote for a clearly noted readme/changelog entry noting the change.

**Comment by benyanke** [comment]
Only other thing I'd do is perhaps also add a readme change to this PR, making clear the executable name has changed.

**Comment by IsmaelMartinez** [comment]
Ignore the travis failure... this is the "temperamental" stuff that I was talking about.

**Comment by IsmaelMartinez** [comment]
Also, regarding release notes, I tend to write up the closed issues in that (or something similar)

A mention to the name of the executable changing should go at the top of the it.

**Pull request merged**